doc/setup/installing_linux_python.adoc

```diff
@@ -26,27 +45,22 @@ git clone https://github.com/OpenSimulationInterface/open-simulation-interface.g
 cd open-simulation-interface
 ----
 +
-. Create a new virtual environment.
-+
-----
-virtualenv -p python3 venv
-----
-+
-. Activate the virtual environment.
+. Optionally create and activate a new virtual environment.
 +
 ----
+python3 -m venv venv
 source venv/bin/activate
 ----
 +
 . Install Open Simulation Interface.
 .. Local installation
 +
 ----
-python3 -m pip install .
+pip install .
 ----
 +
 .. Global installation
 +
 ----
-sudo pip3 install .
+sudo pip install .
```

Comment:
Do you really sudo?

Reply:
I don't ;), but then I never install globally anyway.

More seriously: That was part of the existing instructions, and yes, if you want the package installed globally then you likely need root permissions on most systems. On Ubuntu e.g. if you do not have root permissions, it will just install for the user.

Not that I would recommend anything like that, hence the local installation is the first suggested one.